src/test/java/nextstep/courses/domain/ImageSizeTest.java

```diff
@@ -0,0 +1,27 @@
+package nextstep.courses.domain;
```

Comment:
테스트 코드도 프로덕션 코드의 패키지 구조와 동일하게 맞춰 주시면 좋을 것 같아요
```suggestion
package nextstep.courses.domain.image;
```